src/lib/native.ts

```diff
@@ -34,11 +33,11 @@ export class AutoLauncher {
     this.enabled = result
     return result
   }
-  async enable(): Promise<any> {
+  async enable(): Promise<void | ChildReturnData> {
```

Comment:
Honestly I'm not using the stdout/stderr right now in the code when calling these functions but I figured in the future I may want to throw an error or status update based on that data. Sometimes there can be a failure ( thought I haven't seen that in my testing) in which case it would be good to be able to detect that and alert the user. This code was improved already in the #13 branch so I won't modify it here.